internal/accounts/create.go

```diff
@@ -134,3 +147,301 @@ func create(
 		include: createFlags.Include,
 	}, nil
 }
+
+func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow.Account, error) {
+	log := output.NewStdoutLogger(output.InfoLog)
+
+	log.Info(fmt.Sprintf("%s What name would you like to give this new account?", output.GoEmoji()))
+	name := output.NamePrompt()
+
+	networkName, selectedNetwork := output.CreateAccountNetworkPrompt()
+	// create new gateway based on chosen network
+	gw, err := gateway.NewGrpcGateway(selectedNetwork.Host)
+	if err != nil {
+		return nil, err
+	}
+	privateJsonFileName := ""
+	saveToGitIgnore := false
+	if selectedNetwork != config.DefaultEmulatorNetwork() {
+		privateJsonFileName = fmt.Sprintf("%s.private.json", name)
+		log.Info(fmt.Sprintf("%s For security purposes, the private key generated for this account will be "+
+			"stored in separate file: %s", output.WarningEmoji(), privateJsonFileName))
+		saveToGitIgnore = output.AddToGitIgnorePrompt(privateJsonFileName)
+	}
+	log.Info(fmt.Sprintf("%s This command will perform the following:\n", output.WarningEmoji()))
```

Comment:
I would drop the new lines, and just add it after the list, it makes the output nicer:
```
✔ Local Emulator

This command will perform the following:
- Generate a new ECDSA P-256 public and private key pair
- Create a new account on Local Emulator and pair the public key with the new account
- Save the newly created account configuration to flow.json
```

instead of:
```
✔ Local Emulator
⚠️ This command will perform the following:

- Generate a new ECDSA P-256 public and private key pair

- Create a new account on Local Emulator and pair the public key with the new account

- Save the newly created account configuration to flow.json

Use the arrow keys to navigate: ↓ ↑ → ←
```

Reply:
addressed